server/src/model/family.py

```diff
@@ -0,0 +1,41 @@
+"""MongoDb family model."""
+# 'annotations' import needs to return enclosing type:
+# https://www.python.org/dev/peps/pep-0563/
+from __future__ import annotations
+
+from typing import Optional, Tuple, cast
+
+from mongoengine import Document, IntField, ListField, ObjectIdField, StringField
+from src.utils.error import AppError, ErrorCode, error_bounded
+
+
+class Family(Document):
+    """Family model class that stores ID."""
+
+    sync_interval_days = IntField(required=True)
+    name = StringField(required=True)
+    mate_ids = ListField(ObjectIdField(), default=list)
+
+    meta = {"collection": "families", "strict": False}
+
+    def get_id(self) -> str:
+        """Return string version of mongo oid."""
+        return str(self.id)
+
+    @staticmethod
+    @error_bounded(None)
```

Comment:
Error bounding with none?

Reply:
I didn't think about that, I was copying from here: https://github.com/SyncM8/syncm8/blob/main/server/src/model/user.py#L64-L70

Should we convert all of these lookup error bounds to `ErrorCode.MONGO_ERROR`?

Reply:
Ya, I think that's a good idea.

Reply:
Ya, lets not hide errors like this. 
Also, we can make another ticket to fix the old ones so as to not increase the scope of this pr.

Reply:
sounds good, we can discuss the specifics in the future for better error handling different scenarios.